frontend/src/components/button/CatButton.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+
+type CatButtonProps = {
+    onClickFunction: () => void;
+    width: number;
+    height: number;
+};
+
+const CatButton = ({ onClickFunction, width, height }: CatButtonProps) => {
+  return (
+    <button>
+      <Image src="/assets/catFootprint.png" alt="catFootprint" onClick={onClickFunction} width={width ?? 42} height={height ?? 42} />
```

Comment:
```suggestion
      <Image src="/assets/cat-footprint.png" alt="catFootprint" onClick={onClickFunction} width={width ?? 42} height={height ?? 42} />
```
url 은 kebab case 가 더 자연스럽지 않나요?